README.md

```diff
@@ -3,6 +3,20 @@
 A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI)
 API to add native thread support.
 
+> __NOTE__: this proposal is considered a legacy proposal, retained for engines
+> that can only support WASI v0.1 (`preview1`). After much debate, future work
+> on threads will happen in the [shared-everything-threads] proposal which adds
+> component model [built-ins] for thread spawning, among other things. The goal
+> is that WASI v0.2 and following will use [shared-everything-threads] (once
+> fully implemented) and this proposal can eventually be removed. In the
+> meantime, users experimenting with this proposal can continue to get help
+> with questions and bugs by opening issues on this repository and tagging
+> various WAMR maintainers who plan to continue supporting WASI v0.1 (e.g.,
+> @loganek, @yamt, @wenyongh).
```

Comment:
a question; do other runtimes like wasmtime and wasmer plan to drop wasi-threads anytime soon?

Reply:
Not sure about Wasmer (@syrusakbary?) but I don't see Wasmtime doing so while there's still preview1-ABI support in tree. And that won't go away for a while (a long while?) because there needs to be some transition time between preview1 and preview2+.

Reply:
then it seems a bit awkward to redirect wasi-threads users to WAMR maintainers here.

Reply:
Oh, I see what you mean. I guess it's mostly WAMR maintainers that have motivation to keep this proposal well supported. But perhaps we should just drop the "WAMR" part...

Reply:
maybe a list of known implementations can be helpful to users?
working implementations i'm aware of: toywasm, wasmer, wasmtime, wamr.
i guess the list won't grow anymore.

Reply:
Ok, let's discuss in https://github.com/WebAssembly/wasi-threads/pull/51.